source/codegen/metadata/niscope/functions.py

```diff
@@ -277,7 +277,10 @@
                 'type':'ViInt32'
             }
         ],
-        'returns':'ViStatus'
+        'returns':'ViStatus',
+        'proto_values': {
```

Comment:
Actually, the reason I had added separate `proto_values` dictionary at function level was for easy lookup of "field names" in our codegen to check if the field should be assigned custom field number. The key for entries in dictionary was name of the proto field which should be assigned custom field number, so we would have entries like `value`, `value_raw`, `value_mapped` etc (`value` being name of the field in this specific case).

That said, I think keeping that information with parameter is fine too. I will add `grpc_field_number` attribute on all of these functions I had modified earlier. `grpc_raw_field_number` need not be explicitly mentioned in these cases, since they will automatically get next incremental number.